sys/oom_unix_test.go

```diff
@@ -77,6 +77,8 @@ func adjustOom(adjustment int) (int, error) {
 		return 0, err
 	}
 
+	defer cmd.Process.Kill()
```

Comment:
put it after `cmd.Start`? if error is from `WaitForPid`

Reply:
done